lua/toggleterm.lua

```diff
@@ -248,6 +256,9 @@ end
 --- @param size number
 local function toggle_nth_term(num, size)
   local term = find_term(num)
+
+  update_origin_win(term.window)
+
   if find_window(term.window) then
```

Comment:
`update_origin_win` is here because `find_window` jumps to terminal window - so `origin_win` will be lost in `close` function.